shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/tcl/SetTransactionStatement.java

```diff
@@ -17,10 +17,25 @@
 
 package org.apache.shardingsphere.sql.parser.sql.common.statement.tcl;
 
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.constant.Scope;
+import org.apache.shardingsphere.sql.parser.sql.common.constant.TransactionAccessType;
+import org.apache.shardingsphere.sql.parser.sql.common.constant.TransactionIsolationLevel;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
 
 /**
  * Set transaction statement.
  */
+@Getter
+@Setter
```

Comment:
Is it possible to use final and @RequiredArgsConstructor to instead of getter and setter?

Reply:
> Is it possible to use final and @RequiredArgsConstructor to instead of getter and setter?

I think we can use `setter`, `getter`, because those properties are not required, maybe only part of them.